### ISSUE TITLE
module 'pcapkit.utilities' has no attribute 'stacklevel'

### ISSUE THREAD

**Issue by gghez** [JarryShaw/PyPCAPKit#58]
**Describe the bug**

Cannot run cli or import pcapkit in python script.

```
Traceback (most recent call last):
  File "./venv/bin/pcapkit-cli", line 5, in <module>
    from pcapkit.__main__ import main
AttributeError: module 'pcapkit.utilities' has no attribute 'stacklevel'
```

**System information**

 - OS Version: macOS Mojave 10.15.6
 - Python Version: 3.8.5
 - Python Implementation: CPython

**Traceback stack**

```
/Users/ohoh/hehe/venv/lib/python3.8/site-packages/pcapkit/__init__.py:0: DevModeWarning: development mode enabled
  PCAPKIT_DEVMODE=true
Traceback (most recent call last):
  File "./venv/bin/pcapkit-cli", line 5, in <module>
    from pcapkit.__main__ import main
  File "/Users/ohoh/hehe/venv/lib/python3.8/site-packages/pcapkit/__init__.py", line 82, in <module>
    import pcapkit.all
  File "/Users/ohoh/hehe/venv/lib/python3.8/site-packages/pcapkit/all.py", line 20, in <module>
    from pcapkit.utilities import *  # pylint: disable=redefined-builtin
AttributeError: module 'pcapkit.utilities' has no attribute 'stacklevel'
```

**Expected behavior**

It works

**Additional context**

Only `pip install pypcapkit dpkt scapy pyshark` worked. All other with brackets reported a "no matches found: pypcapkit[xxx]"

**Comment by JarryShaw** [comment]
`AttributeError` has been fixed in version **0.15.4**.

As to `pypcapkit[xxx]`, i have tested on both pip and pipenv install commands, they're working properly. Could you please provide more information about the error?

**Comment by JarryShaw** [comment]
Closing #58 due to lack of response for long time.
